codegen/core/src/main/scala/com/lightbend/akkasls/codegen/ModelBuilder.scala

```diff
@@ -297,6 +301,13 @@ object ModelBuilder {
             Some(ActionService(serviceName, commands))
           case ServiceType.SERVICE_TYPE_VIEW =>
             val methodDetails = methods.flatMap { method =>
+              println(
+                s"# ${serviceName.protoName} method ${method.getName} [${method
+                  .getOptions()
+                  .getExtension(com.akkaserverless.Annotations.method)
+                  .getView()
+                  .hasUpdate}] [${method.getOptions().getExtension(com.akkaserverless.Annotations.method).getView().hasQuery}]"
+              ) // FIXME
```

Comment:
Added `Annotations.registerAllExtensions`, but there is still something strange. Looking at the debug output from this, it seems like the CustomerByName is parsed twice, and second time it has false on the `hasUpdate`

```
[DEBUG] Looking at descriptor customer/view/customer_view.proto
# CustomerByName method UpdateCustomer [true] [false]
# CustomerByName method GetCustomers [false] [true]
# CustomerByEmail method UpdateCustomer [true] [false]
# CustomerByEmail method GetCustomer [false] [true]
# CustomerSummaryByName method GetCustomers [false] [true]
# CustomerSummaryByName method UpdateCustomer [true] [false]
# CustomersResponseByName method GetCustomers [false] [true]
# CustomersResponseByName method UpdateCustomer [true] [false]
[DEBUG] Raw value entity name:
[DEBUG] Raw replicated entity name:
[DEBUG] Looking at descriptor customer/action/customer_action.proto
[DEBUG] Raw value entity name:
[DEBUG] Raw replicated entity name:
[DEBUG] Looking at descriptor customer/api/customer_api.proto
[DEBUG] Raw value entity name:
[DEBUG] Raw replicated entity name:
[DEBUG] Looking at descriptor customer/domain/customer_domain.proto
[DEBUG] Raw value entity name: CustomerValueEntity
[DEBUG] Raw replicated entity name:
[DEBUG] Looking at descriptor customer/view/customer_view.proto
# CustomerByName method UpdateCustomer [false] [false]
# CustomerByName method GetCustomers [false] [false]
```